datasets/ecco-surface-height-change.data.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: ecco-surface-height-change
+name: "ECCO sea-surface height change from 1992 to 2017"
+description: "Gridded global sea-surface height change from 1992 to 2017 from the Estimating the Circulation and Climate of the Ocean (ECCO) ocean state estimate."
+media:
+  src: ::file ./ecco-surface-height-change--dataset-cover.jpg
```

Comment:
With 1.8MB this image is pretty big. Consider cutting it down to 1920 on the wide side and running it through http://compressor.io/